src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
+$main-accent-color: #060b35;
+$secondary-accent-color: #616070;
+$blue-accent-color: #00acdc;
+$white-accent-color: #fff;
+$font-family: Roboto, sans-serif;
+
+@import '../utils/veriables';
```

Comment:
There is a typo in the import statement: `@import '../utils/veriables';`. It should be `@import '../utils/variables';` to correctly import the file. Ensure that the path and filename are correct.